docs/build-n-test.md

```diff
@@ -132,6 +132,12 @@ Success.
         [ Verified ] 
 ```
 
+## Static Code Analysis
+### CppCheck
+```
+$ make cppcheck
```

Comment:
We shall not specify the target which is mapped to exact tool names. Instead, the build target might be `static-analysis`.

Reply:
My original design is to have two target, one is for `CppCheck` and one is for `Infer`.
I think I will modify the target to `static-analysis`, and it would use `CppCheck` and `Infer` to do the static code analysis.

Reply:
Done.